### PR TITLE
[ISSUE #4547] Add unit test for OpenFunctionSourceConnector.

### LINKED ISSUE #4547: [Unit Test] Add unit test for OpenFunctionSourceConnector.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Read the unit testing guidelines

- [X] I have read.

### Unit test request

Add unit test for OpenFunctionSourceConnector.

### Describe the unit tests you want to do

Add unit test for OpenFunctionSourceConnector.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by VishalMCF**
Fixes #4547 .

### Motivation

OpenFunctionConnector module was lacking test cases.

### Modifications

Added test file for OpenFunction Source Connector. Explored SpringSourceConnector and based on that the changes were added.

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by VishalMCF** [comment]
@yanrongzhen Please review the pull request. If it is okay based on that I will add changes for OpenFunctionSinkConnectorTest

File: eventmesh-connectors/eventmesh-connector-openfunction/src/test/java/org/apache/eventmesh/connector/openfunction/source/connector/OpenFunctionSourceConnectorTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class OpenFunctionSourceConnectorTest {
+
+    @Spy
+    private OpenFunctionSourceConnector connector;
```
[the remark this answers is not in the document]
Comment (VishalMCF) [reply eventmesh-connectors/eventmesh-connector-openfunction/src/test/java/org/apache/eventmesh/connector/openfunction/source/connector/OpenFunctionSourceConnectorTest.java]: @pandaapo I have made the changes as per your previous comment.

**Review by mxsm — CHANGES_REQUESTED** [review]
@VishalMCF  Please fix CI error

**Comment by VishalMCF** [comment]
@mxsm Is it possible to see the test.html file to see what errors are happening? I have imported the checkstyle.xml and when I run ./gradlew check it works fine in my system.

**Comment by VishalMCF** [comment]
@mxsm I have made the requested changes.

**Review by xwm1992 — APPROVED** [review]

**Review by mxsm — APPROVED** [review]
LGTM

**Pull request merged**